pandas/core/generic.py

```diff
@@ -2055,6 +2055,37 @@ def drop(self, labels, axis=0, level=None, inplace=False, errors='raise'):
         Returns
         -------
         dropped : type of caller
+
+        Examples
+        --------
+        >>> df = pd.DataFrame([[np.nan, 2, np.nan, 0],
+        ...                    [3, 4, np.nan, 1],
+        ...                    [np.nan, np.nan, np.nan, 5],
+        ...                    [np.nan, 3, np.nan, 4]
+        ...                   ],
```

Comment:
I would just use a Frame with numbers (without the NaNs), the NaNs are not really an added value to the example